.github/workflows/build_portable_linux_python_packages.yml

```diff
@@ -57,6 +64,7 @@ jobs:
           IS_PR_FROM_FORK: ${{ github.event.pull_request.head.repo.fork }}
```

Comment:
We might be able to drop this environment variable altogether, and instead use `github.run_id` from https://docs.github.com/en/actions/reference/workflows-and-actions/contexts to have the script continue to do its own lookup. I don't like introspecting the current workflow run if we can avoid it though, so keeping this explicit could be useful.

I'll have more of a clear idea once I've connected this workflow to `ci.yml`.